specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -65,6 +65,52 @@
         }
       }
     },
+    "/providers/Microsoft.Elastic/listElasticVersions": {
+      "get": {
+        "tags": [
+          "ElasticVersions"
+        ],
+        "summary": "Get a list of available versions for a region.",
+        "operationId": "ElasticVersions_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "region",
+            "description": "Region where monitor resource deployment will took place.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "subscriptionId",
+            "description": "Subscription ID where monitor resource will be deployed.",
```

Comment:
>Subscription ID where monitor resource will be deployed [](http://example.com/codeflow?start=28&length=55)

Subscription ID where monitor resource is deployed

Reply:
Updated